router/batchrouter/batchrouterBenchmark_test.go

```diff
@@ -39,3 +44,68 @@ func Benchmark_GetStorageDateFormat(b *testing.B) {
 func randomString() string {
 	return strings.ReplaceAll(uuid.Must(uuid.NewV4()).String(), "-", "")
 }
+
+func Benchmark_JSONUmarshal(b *testing.B) {
+	config.Load()
```

Comment:
Do we still need the `config.Load()`? :thinking:

Reply:
This PR is based on the `release/1.1.x` branch